book/src/parachains/composable-parachain-overview.md

```diff
@@ -0,0 +1,33 @@
+
+The Composable Parachain does not use a traditional bridge. Instead, Mosaic serves as a liquidity and transfer availability layer that is able to facilitate asset transfer between chains and layers, irrespective of size, in a safe and secure manner. Mosaic accomplishes this through a process of forecasting, and dynamic liquidity re-balancing. Mosaic is designed to be the first EVM <> Substrate bridge to support the transfer of assets across several ecosystems whilst avoiding failures in cross-chain transactions. 
+
+The Composable Parachain will also integrate Centauri, our Substrate–IBC bridge to support cross-ecosystem communication between the Cosmos and Polkadot ecosystems. 
+
+
+![composable_parachain_architecture](./composable-parachain-architecture.png)
+
+
+
+## How the Composable Parachain synergizes with Picasso?
+
+Within the Composable ecosystem, we have established a parachain on Kusama, known as Picasso and on Polkadot, known as Composable Parachain. The idea for a dual-parachain narrative came about when we realized crypto networks are designed without a stable launch process. As such they rely on testnets for stability and performance testing. In the case of the DotSama ecosystem, Kusama is treated as such a testnet even though its capabilities and potential span beyond simply being a testnet for Polkadot. 
+
+As the canary network of the Composable Parachain, Picasso serves as the development and deployment hub for modular dApps. Developers on Picasso can leverage our [interoperable infrastructure](https://medium.com/composable-finance/understanding-composables-modularly-interoperable-infrastructure-e3986fa58d21) to streamline the development process while Composable Labs provides grants and technical support to further enable aspiring founders to launch innovative pallets on Picasso. As such, Picasso acts as a springboard for the development and deployment of pallets that are natively cross-chain. 
+
+Identical pallets on the two separate chains will communicate with one another using IBC or XCM depending on which is available at the time and has the most efficient performance. As a result, both parachains will remain up-to-date and consistent with one another in real-time enabling seamless communication and transaction matching between pallets across networks. This new dual parachain approach means users can operate the parachains in concert with one another, resulting in maximized efficiency instead of any redundant use of pallets. 
+
+We designed our strategy to allow pallets, builders, and ultimately end-users, to leverage both parachains simultaneously to promote the stability and increased efficiency of cross-chain functions. The combination of available liquidity from the Polkadot and Kusama ecosystems is intended to greatly increase the earning power of both the Composable and Picasso networks. This would also, in theory, increase the capital efficiency of both parachains.
```

Comment:
```suggestion
We designed our strategy to allow pallets, builders, and ultimately end-users, to leverage both parachains simultaneously to ensure the stability and increased efficiency of cross-chain functions. This new dual parachain approach means users can operate the parachains in concert with one another, resulting in maximized efficiency instead of any redundant use of pallets. Thus, Composable becomes the **contract chain**, and Picasso the **pallets chain**.  The combination of available liquidity from the Polkadot and Kusama ecosystems would greatly increase the earning power of users within the Composable and Picasso networks. This would also, in theory, increase the capital efficiency of both parachains.
```